ekumenlabs/fruit_detection
# Add detection sample

Goes on top of #5 

This PR includes a demo face detection application which completes the routing of this component. We will later change the classifier and detector by the fruit detector keeping this interface.


To test it:

```
docker compose -f docker/docker-compose.yml --profile test_real_pipeline build
docker compose -f docker/docker-compose.yml --profile test_real_pipeline up
```

Make sure you are in front of the camera and look at the RQt window. You should see two images, on the right the detection with a green box. Try selecting the detection topic in the topic monitor and observe how the bounding box size changes as you move. Same withe center.
